CHANGELOG.md

```diff
@@ -13,6 +19,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
   stopping, checkpointing, etc.
 - Add top-level base class for pickling TensorFlow models.
 - Add partial fitting functionality for the MLP.
+- Add support for missing labels for MLPClassifier.
```

Comment:
maybe this would be clearer with "missing labels during multilabel classification".   having missing labels in a typical classifier doesn't really make sense (right?), so this might confuse people as is